ext/calendar/calendar.c

```diff
@@ -218,7 +218,7 @@ enum { CAL_MONTH_GREGORIAN_SHORT, CAL_MONTH_GREGORIAN_LONG,
 };
 
 /* for heb_number_to_chars */
-static char alef_bet[25] = "0����������������������";
+static char alef_bet[25] = "0����������������������";
```

Comment:
ISO-8859 to UTF-8 conversion. This file is otherwise encoded in ISO-8859 because to make it the UTF-8 there should be some more changes done in the C code itself... Haven't got to that part yet how to do it properly so for now maybe ISO-8859 should stay I think.

Reply:
@cmb69 addressed and fixed as suggested via https://github.com/php/php-src/pull/3872